messaging/app/src/main/java/com/google/firebase/quickstart/fcm/kotlin/MainActivity.kt

```diff
@@ -8,10 +8,14 @@ import android.util.Log
 import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import com.google.android.gms.tasks.OnCompleteListener
+import com.google.android.gms.tasks.Task
+import com.google.firebase.functions.FirebaseFunctions
```

Comment:
If the comment above gets applied, we'll need to import functions ktx too.
```suggestion
import com.google.firebase.functions.FirebaseFunctions
import com.google.firebase.functions.ktx.functions
```